# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../iasl/iasl-20110413-Makefile.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- acpica-unix-20110413/compiler/Makefile.orig	2011-05-14 00:23:37.064002301 +0200
+++ acpica-unix-20110413/compiler/Makefile	2011-05-14 00:47:05.112002302 +0200
@@ -67,11 +67,11 @@
 	asluuid.o \
 	aslwalks.o \
 	dtcompile.o \
+	dtparserparse.o \
+	dtparserlex.o \
 	dtexpress.o \
 	dtfield.o \
 	dtio.o \
-	dtparserlex.o \
-	dtparserparse.o \
 	dtsubtable.o \
 	dttable.o \
 	dttemplate.o \
@@ -166,11 +166,14 @@
 
 INTERMEDIATES = \
 	aslcompilerlex.c \
-	aslcompilerparse.c
+	aslcompilerparse.c \
+	dtparserparse.c
 
 MISC = \
 	aslcompiler.y.h \
-	aslcompilerparse.output
+	aslcompilerparse.output \
+	dtparser.y.h \
+	dtparserparse.output
 
 
 #
